exdb/tests/browser_tests.py

```diff
@@ -187,8 +215,47 @@ def instrument_istanbul(cls):
 class DefaultLiveServerTestCase(StaticLiveServerTestCase):
     running_total = IstanbulCoverage()
 
+    @classmethod
+    def setUpClass(cls):
+        if CustomRunner.skip_browser_tests:
+            raise SkipTest('Skipped due to argument')  # pragma: no cover
+        super(DefaultLiveServerTestCase, cls).setUpClass()
+
+    class selenium_client:
+
+        def __init__(self, driver):
+            self.driver = driver
+
+        def get(self, url):
+            self.driver.get(CustomRunner.live_server_url + url)
+
+        def force_login(self):
+            'Login a browser without visiting the login page'
+            c = Client()
+            # avoid setting the password and force_login for speed
+            user_object = get_user_model().objects.create(username='user')
+            c.force_login(user_object)
+            if CustomRunner.live_server_url not in self.driver.current_url:
+                # if we would be trying to set a cross domain cookie change the domain
+                self.get(reverse('login'))
+
+            cookie = {'name': 'sessionid', 'value': c.session.session_key}
+            try:
+                self.driver.add_cookie(cookie)
+            except selenium.common.exceptions.WebDriverException:
```

Comment:
It is covered if you run it with an updated version of phantomjs. Shane had the same issue. I was thinking that we should leave it uncovered in case the bug is fixed so that we can remove this altogether.